ktcacheorbust.cc

```diff
@@ -180,7 +214,16 @@ bool CacheOrBust::Worker::do_get(
     delete[] data;
   } else {
     _opcounts[tid][MISS]++;
-    sess->printf("END\r\n", key.c_str());
+
+    std::string url;
+    if (tokens.size() >= 3)
```

Comment:
I 50% feel like these sizechecks should be against globally-defined variables that refer to exact roles/reasons - it's going to be easy to miss a number change in some location if one ever needs to update this code

Reply:
I 50% agree.  But I'm not going to do that today :)